<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu_principal</class>
 <widget class="QMainWindow" name="menu_principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>440</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>385</width>
    <height>440</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>385</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Games Central</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/game.png</normaloff>../img/game.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(40, 42, 58);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_hangman">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fredoka One</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(76, 218, 99);
border-radius: 8px;
color: white;</string>
    </property>
    <property name="text">
     <string>Hangman Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_tictactoe">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fredoka One</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 46, 87);
border-radius: 8px;
color: white;</string>
    </property>
    <property name="text">
     <string>Tic Tac Toe</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_games">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fredoka One</family>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #4CDA63;
background-color: transparent;</string>
    </property>
    <property name="text">
     <string>GAMES</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_central">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>77</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fredoka One</family>
      <pointsize>22</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: transparent;</string>
    </property>
    <property name="text">
     <string>central</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_jokenpo">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fredoka One</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 150, 1);
border-radius: 8px;
color: white;</string>
    </property>
    <property name="text">
     <string>JokenPo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_memory">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>350</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fredoka One</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(1, 122, 255);
border-radius: 8px;
color: white;</string>
    </property>
    <property name="text">
     <string>Memory Game</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
